docs/content/node-operation/access-node-setup.mdx

```diff
@@ -1,22 +1,327 @@
 ---
 title: Setting Up a Flow Access Node
 sidebar_title: Access Node Setup
-description: How to run a Permissionless Access node
+description: How to run a permissionless Access node
 ---
 
-This guide is for running a Permissonless Access node on Flow. If you are planning to run a different type of staked node then see [node bootstrap](node-bootstrap.mdx).
+This guide is for running a permissonless Access node on Flow. If you are planning to run a different type of staked node then see [node bootstrap](node-bootstrap.mdx).
+
+Permissionless Access nodes allow any operator to run a Flow Access node.
+Unlike the other staked nodes, a permissionless access node does not have to be approved by the service account before it can join the network, hence the term "permissionless". The goal is to make all node types permissionless and this is the first step towards achieving that goal.
+
+## Who should run a Permissionless Access node?
+dApp developers can choose to run their own private permissionless access node and move away from using the community access nodes. This will also allow them to not be subjected to the API rate limits of the public access nodes.
+
+Node operators can also run their own permissionless access node and provide access to that node as a service.
+
+Chain analytics, audit and exploration applications can run such an access node and do not have to rely on third parties for the state of the network.
+
+## Limitations
+This is a preliminary implementation of full permissionless node operation and there will be exactly **five** slots in total that will be opened for access nodes.
+If there are more than five staked candidate access nodes at the end of the staking phase on 14th Feb 2023 EOD PT, five of those will be selected by an on-chain random selection process and no other permissionless ANs can join until others unstake in a future epoch.
+
+More slots will be opened in the future if this goes well.
+
+Support for unlimited access nodes requires staking auctions to be implemented, which is still an upgrade planned for the future.
+
+## How to run a Permissionless Access node?
+
+> Note: To run an access node you will need to provision a machine or virtual machine to run your node software. Please follow the [node-provisioning](node-provisioning.mdx) guide for it.
+You can provision the machine before or after your node has been chosen.
+
+At a high level, to run a permissionless Access node, you will have to do the following steps:
+1. Generate the node identity (private and public keys, node ID etc.).
+2. Stake the node with 100 FLOW by **EOD Tuesday, 14th February 2023 Pacific time (02/14/2023 11:59 PM or 02/15/2023 7:59 AM UTC)** (see [timing](#timing)) by providing the node information generated in step 1.
```

Comment:
I feel like the Feb 14 date should go out in an announcement, whereas the documentation should be more general and say something like "each Tuesday EOD". Otherwise it will be outdated almost immediately. 

We could say here that the _first_ slots will open Feb 14.

By the way, do we provide a way for people to look-up whether there are any slots available? After the initial few allocations of slots where we announce them explicitly, a look-up tool/script would be the way people check for available slots.

Reply:
make sense - lemme reword it

Reply:
ok - i made the guide more generic but called out the exact time inside the limitations section.